services/frontend/src/scenes/ProjectDetails/ProjectDetails.scss

```diff
@@ -44,6 +44,7 @@
     margin: 0 $default-spacing*4;
     padding-top: $default-spacing;
 
+
```

Comment:
Delete extra line (I think this is only not a linter error because of it being an scss file).